patches/chrome-browser-ui-bookmarks-bookmark_bar_controller.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/bookmarks/bookmark_bar_controller.cc b/chrome/browser/ui/bookmarks/bookmark_bar_controller.cc
+index 77fec75f06f7ad02840ce764da2ef984cb0f6346..88468ae3dabee435b91c957aa87c050a6d84fcc0 100644
+--- a/chrome/browser/ui/bookmarks/bookmark_bar_controller.cc
++++ b/chrome/browser/ui/bookmarks/bookmark_bar_controller.cc
+@@ -161,6 +161,7 @@ bool BookmarkBarController::ShouldShowBookmarkBar() const {
+   if (tabs.empty()) {
+     return false;
+   }
++  BRAVE_BOOKMARK_BAR_CONTROLLER_SHOULD_SHOW_BOOKMARK_BAR
```

Comment:
`#define GetForegroundTabs() GetForegroundTabs(); return IsShowingNTP(active_tab->GetContents()) && prefs->GetBoolean(kAlwaysShowBookmarkBarOnNTP)` ?

I want to make sure something will catch this though if `ShouldShowBookmarkBar` changes. cc @cdesouza-chromium

Reply:
I'm not sure I understand what you are saying.

Reply:
discussed in DM, but this has already changed and I commented on the lastest version